src/app/features/ledger/utils/bitcoin-ledger-utils.ts

```diff
@@ -1,5 +1,7 @@
 import Transport from '@ledgerhq/hw-transport-webusb';
+import { Psbt } from 'bitcoinjs-lib';
```

Comment:
Why use this library?

Reply:
I didn't get it to work with the signer library. Would love to remove but not biggest priority.